### DISCUSSION TITLE
Оставить расу скелетов донатерам на основу, но с большими изменениями.

### DISCUSSION THREAD

**Discussion by kaworu177** [space-sunrise/space-station-14#599] [Ideas]
Я считаю, что скелетов нужно оставить на основу донатерам, но с большими изменениями: 

**ИЗМЕНЕНИЯ**

1. Теперь, cкелеты получают урон от давления, но немного меньше, чем обычный гуманоид, так-же нужно понизить скелету порок получения урона от давления
2. Теперь, скелеты обязаны пить молоко, но очень редко за смену (1 раз за час), Так-же, скелет может вместо молока употреблять сливки и любые коктейли, в которые входят сливки. У скелетов всё-ещё  нету голода, и потребности к пище
3. Изменение резистов - Понизить почти ВСЕ резисты, до уровня "Чуть лучше троттин", так-как сейчас скелеты - Мета, которая сильнее любой расы.

**Ивент: Скелет в шкафу** 

Ивент всё-ещё останется, но шансы на появление будут снижены.
Остальное без изменений

**Вики изменения**

Насчёт этого - ничего не скажу, нужно будет написать лор, но в этом я не силач

**Зачем это нужно?**

Для повышение покупок спонсорки. (Деньги -  ура).

Надеюсь на рассмотрение идеи.

**Comment by Rinary1** [comment]
Анлак, отказано, скелеты это ивентовая расса. Так-же они являються антагонистами(вне праздника или ивента)
